webextensions/api/bookmarks.json

```
@@ -7,7 +7,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/BookmarkTreeNode",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "3"
```

:x: https://github.com/chromium/chromium/commit/acaf52152973b07a7348a672f3c72b4184493a51 in [4.0.212.1](https://github.com/chromium/chromium/releases/tag/4.0.212.1) before 4.0.223.

```suggestion
                "version_added": "4"
```